content/en/about-xlt/_index.md

```diff
@@ -13,26 +13,12 @@ menu:
     weight: 1
 ---
 
-## Introduction
-XLT (Xceptance Load Test) is a multi-purpose load and performance test tool with the ability to also be used for test automation. The test automation piece supports WebDriver-style automation but adds features to it. Load testing and test automation together permit you to run client performance testing. That means, you can use a real browser to run a performance test at scale rather than trying to build all the calls manually.
+XLT (Xceptance Load Test) is a multi-purpose load and performance test tool with the ability to be used for test automation too. The test automation supports WebDriver-style automation but adds features to it. Load testing and test automation together permit you to run client performance testing. That means, you can use a real browser to run a performance test at scale rather than trying to build all the calls manually.
 
-XLT is powered by Java and hence it runs almost everywhere. Java also enables you to program your scripts rather than just configuring it. That gives you the power of a full programming languages at you disposal and of course, you can use many 
+XLT is powered by Java and hence it runs almost everywhere. Java also enables you to program your scripts rather than just configuring them. That gives you the power of a full programming languages at you disposal and of course, you can use almost any Java library as part of XLT.
 
 Make sure you read about the [history](history) of XLT.
 
-## About "XLT in the Nutshell"
-The documentation section "XLT in the Nutshell" is supposed to give you a big picture introduction. It talks about features and functionalities as well as points you to the how-tos and manual entries that deal with them in detail.
+The documentation section "About XLT - In a Nutshell" gives you a big picture introduction. It talks about features and functionalities as well as points you to the how-tos and manual entries that deal with them in detail.
 
-In contrast, the manual might refer back to this nutshell section to explain higher level concepts and terms.
-
-## Market
-
-## Load and Performance
-
-
-## Test Automation
-
-## Client Performance Testing
-
-## License
-XLT is open source under [Apache License 2.0](https://opensource.org/licenses/Apache-2.0). 
```

Comment:
Actually this was the place where I'd have put and expected a hint (and link) to XLT's license...

Reply:
I know and I wonder why my merge went wrong. Checking

Reply:
Ok, I know... because I removed the sub sections and let the part be assembled automatically it went away. I pushed an update with an extra license page.